src/commcare_cloud/fab/operations/release.py

```diff
@@ -287,18 +287,12 @@ def _update_virtualenv(virtualenv_current, virtualenv_root, requirements):
                      user=env.sudo_user)
                 pip_install(cmd_prefix, timeout=60, quiet=True, proxy=env.http_proxy, requirements=[
                     posixpath.join(requirements, 'prod-requirements.txt'),
-                    posixpath.join(requirements, 'requirements.txt'),
```

Comment:
This is already removed from master: https://github.com/dimagi/commcare-cloud/pull/3190

Reply:
Somehow I thought that `control/init.sh` pulled from master, but now that I check, it doesn't. My code was pretty out of date.

I reviewed that PR, but when I found that line in my `fab/operations/release.py` (believing that my code had just been updated) I thought that PR had removed it from a different file, and I was just reproducing the same change in this file.